conda-forge.yml

```diff
@@ -1,3 +1,5 @@
+azure:
+  free_disk_space: true
```

Comment:
Here's the [docs on `azure/free_disk_space`]( https://conda-forge.org/docs/maintainer/conda_forge_yml.html#azure-config )